### ISSUE TITLE
Pods not deleted in case of network issues

### ISSUE THREAD

**Issue by vania-pooh** [aerokube/moon#89]
Getting 504 in tests. Moon logs:
```
vania-pooh:~$ grep 86e526b830c  ~/moon.log
2018/11/14 10:44:35 [CREATING_POD] [browsers] [10.20.32.49] [chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c]
2018/11/14 10:44:35 [CREATING_SERVICE] [browsers] [10.20.32.49] [chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c]
2018/11/14 10:44:35 [POD_CREATED] [browsers] [10.20.32.49] [chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c] [0.02s]
2018/11/14 10:44:35 [SERVICE_CREATED] [browsers] [10.20.32.49] [chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c] [0.02s]
2018/11/14 10:44:35 [AWAITING_SERVICE] [browsers] [10.20.32.49] [chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c]
2018/11/14 10:44:47 [SERVICE_IS_UP] [browsers] [10.20.32.49] [chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c] [12.19s]
2018/11/14 10:44:47 [SESSION_CREATED] [browsers] [10.20.32.49] [chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c] [12.19s]
2018/11/14 10:44:47 http: proxy error: dial tcp: lookup chrome-69-0-f7856a78-38fc-418c-b7e3-086e526b830c on 10.20.64.10:53: server misbehaving
```

Defender logs:
```
2018/11/14 11:59:11 [PROXYING] [POST http://localhost:4444/session]
2018/11/14 11:59:15 [PROXYING] [POST http://localhost:4444/session/8c5dd472cc3819eb57cf0ffc7bb92aaf/url]
2018/11/14 11:59:15 [PROXYING] [POST http://localhost:4444/session/8c5dd472cc3819eb57cf0ffc7bb92aaf/cookie]
2018/11/14 11:59:15 [PROXYING] [DELETE http://localhost:4444/session/8c5dd472cc3819eb57cf0ffc7bb92aaf/cookie]
2018/11/14 11:59:15 [PROXYING] [POST http://localhost:4444/session/8c5dd472cc3819eb57cf0ffc7bb92aaf/elements]
2018/11/14 12:00:15 http: proxy error: context canceled
2018/11/14 12:00:15 http: proxy error: context canceled
2018/11/14 12:00:15 [PROXYING] [GET http://localhost:4444/session/8c5dd472cc3819eb57cf0ffc7bb92aaf/screenshot]
2018/11/14 12:00:15 http: proxy error: context canceled
2018/11/14 12:00:15 [PROXYING] [GET http://localhost:4444/session/8c5dd472cc3819eb57cf0ffc7bb92aaf/screenshot]
2018/11/14 12:00:35 [PROXYING] [DELETE http://localhost:4444/session/8c5dd472cc3819eb57cf0ffc7bb92aaf]
2018/11/14 12:00:55 http: proxy error: context canceled
2018/11/14 12:00:55 http: proxy error: context canceled
2018/11/14 12:00:55 http: proxy error: context canceled
```

**Comment by vania-pooh** [comment]
Could be because of incorrect Ingress timeout settings.

**Comment by vania-pooh** [comment]
Fixed in aandryashin/moon#164, aandryashin/moon#165.
